src/load_map.cpp

```diff
@@ -1180,6 +1180,8 @@ void map_parser::parse_shield_symbolizer(rule & rule, xml_node const& node)
         put(sym, keys::file , parse_path(file));
         optional<halo_rasterizer_e> halo_rasterizer_ = node.get_opt_attr<halo_rasterizer_e>("halo-rasterizer");
         if (halo_rasterizer_) put(sym, keys::halo_rasterizer, halo_rasterizer_enum(*halo_rasterizer_));
+        set_symbolizer_property<symbolizer_base,composite_mode_e>(sym, keys::halo_comp_op, node);
+        set_symbolizer_property<symbolizer_base,transform_type>(sym, keys::halo_transform, node);
```

Comment:
Are these generally missing properties (and not specifically related to your pull request)? If so, could you create a new pull quest and visual tests for them?